src/init.cpp

```diff
@@ -806,6 +806,22 @@ void InitParameterInteraction()
             LogPrintf("%s: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1\n", __func__);
     }
 
+    // This is required by both the wallet and node
+    if (gArgs.IsArgSet("-maxtxfee"))
+    {
+        CAmount nMaxFee = 0;
+        if (!ParseMoney(gArgs.GetArg("-maxtxfee", ""), nMaxFee))
+            InitError(AmountErrMsg("maxtxfee", gArgs.GetArg("-maxtxfee", "")));
```

Comment:
`return InitError` has been changed to just `InitError` here so I think this changes behavior.

Maybe this code should move to the `AppInitParameterInterfaction` function instead of `InitParameterInterfaction` to avoid needing this change.

Reply:
I wasn't sure about that, but `WalletInit::ParameterInteraction` returned a bool and `InitParameterInteraction` returns void, so I removed it. `InitParameterInteraction` looks like it will work, though.

Reply:
> looks like it will work

Did you test that the initialization is aborted when you supply an invalid maxtxfee?

Reply:
> Did you test that the initialization is aborted when you supply an invalid maxtxfee?
It will not abort unless there is a return. I'll move it to `AppInitParameterInteraction` so it can return and abort.